server_admin/topics/authentication/x509.adoc

```diff
@@ -252,6 +277,20 @@ If set, the client certificate trust chain will be always verified at the applic
 
 ==== Adding X.509 Client Certificate Authentication to a Direct Grant Flow
 
+
+. Click *Authentication* in the menu.
+ifeval::[{project_community}==true]
+. Select *Duplicate* from the "Action list" to make a copy of the built-in "Direct grant" flow.
+. Enter a name for the copy.
+. Click *Duplicate*.
+. Click the created flow.
+. Click on the *🗑️* (trash can icon) of the "Username Validation" and click *Delete*.
+. Click on the *🗑️* (trash can icon) of the "Password" and click *Delete*.
```

Comment:
Click the trash can icon 🗑️ of the,,,,

Reply:
Guess you mean Click _on_ the trash can icon